src/jmeter/resources/log4j2.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
what is the use of this file actually? Cos like most of the info are commented out

Reply:
This file is required for generating the `jmeter.log` file when using `jmRun` or `jmReport`.

> Cos like most of the info are commented out

I copied this file over from my Jmeter installation. The comments are present in case you want to change the logging settings. Though it's unlikely that we'll ever change the settings, do you think the comments should still be kept?  It would help avoid googling to try to find the relevant property value, and also gives a list of available properties.